dashboard/utils_test.py

```diff
@@ -877,8 +879,17 @@ def test_get_overall_final_grade_for_course(self):
         FinalGradeFactory.create(user=self.user, course_run=self.crun_fa, passed=True, grade=0.8)
         assert mmtrack.get_overall_final_grade_for_course(finaid_course) == "80"
         ExamRunFactory.create(course=finaid_course)
-        CombinedFinalGrade.objects.create(user=self.user, course=finaid_course, grade="74")
-        assert mmtrack.get_overall_final_grade_for_course(finaid_course) == "74"
+        if before_exam_merge:
+            # if the course run end date is before Fall 2022
+            self.crun_fa.start_date = NEW_COMBINED_FINAL_GRADES_DATE - timedelta(days=1)
+            self.crun_fa.save()
+            CombinedFinalGrade.objects.create(user=self.user, course=finaid_course, grade="74")
+            assert mmtrack.get_overall_final_grade_for_course(finaid_course) == "74"
+        else:
+            self.crun_fa.start_date = NEW_COMBINED_FINAL_GRADES_DATE + timedelta(days=1)
+            self.crun_fa.save()
+            CombinedFinalGrade.objects.create(user=self.user, course=finaid_course, grade="80")
+            assert mmtrack.get_overall_final_grade_for_course(finaid_course) == "80"
```

Comment:
Not sure if these test changes are intentional in this PR because they are out of scope for this.

Reply:
Oh, I think they were to fix a flaky test.